services/galley/src/Galley/Intra/Journal.hs

```diff
@@ -69,12 +70,11 @@ journalEvent typ tid dat tim = view aEnv >>= \mEnv -> for_ mEnv $ \e -> do
 ----------------------------------------------------------------------------
 -- utils
 
-evData :: [TeamMember] -> Maybe Currency.Alpha -> TeamEvent'EventData
-evData mems cur =
+evData :: Natural -> [TeamMember] -> Maybe Currency.Alpha -> TeamEvent'EventData
```

Comment:
not sure if i should have understood this from reading this far in the PR, but what's this member list?  hopefully not the entire team?  would that be worth some haddocks here, or somewhere more appropriate?

Reply:
It is the entire team unfortunately, but it is only required to find whom to send the bill to. It is a task in our TODO list, so we'll take care of it later.